Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -2435,6 +2457,43 @@ void webkit_settings_set_pictograph_font_family(WebKitSettings* settings, const
     g_object_notify_by_pspec(G_OBJECT(settings), sObjProperties[PROP_PICTOGRAPH_FONT_FAMILY]);
 }
 
+/**
+ * webkit_settings_get_math_font_family:
+ * @settings: a #WebKitSettings
+ *
+ * Gets the #WebKitSettings:math-font-family property.
+ *
+ * Returns: The default font family used to display content marked with math font.
+ */
+const gchar* webkit_settings_get_math_font_family(WebKitSettings* settings)
+{
+    g_return_val_if_fail(WEBKIT_IS_SETTINGS(settings), 0);
+
+    return settings->priv->mathFontFamily.data();
+}
+
+/**
+ * webkit_settings_set_math_font_family:
+ * @settings: a #WebKitSettings
+ * @math_font_family: the new default math font family
```

Comment:
If you change it to use nullptr instead of empty string:

```suggestion
 * @math_font_family: (nullable): the new default math font family
```

(Otherwise, we need to add a careful description explaining that empty string is allowed and default.)